contributors/guide/contributor-cheatsheet/README-es.md

```diff
@@ -0,0 +1,393 @@
+que explique por qué no cumple con los requisitos para ser fusionado.
+
+🙂 "Estoy cerrando esta PR porque esta característica no puede admitir el caso de uso X. En 
+su forma propuesta, sería mejor implementarla con la herramienta Y. Gracias por trabajar en esto."
+
+😞 "¿Por qué esto no sigue las convenciones de la API? Esto debería hacerse en otro lugar."
+
+---
+
+## Presentar una Contribución
+
+### Firmar el CLA
+
+Antes de poder presentar una contribución, debes [firmar el Acuerdo de Licencia del 
+Colaborador (CLA)][cla]. 
+El proyecto Kubernetes _solo_ puede aceptar una contribución si tú o tu empresa 
+han firmado el CLA.
+
+Si encuentras problemas al firmar el CLA, sigue 
+las [pautas de solución de problemas del CLA].
+  
+### Abrir y Responder Issues
+
+Los Issues de GitHub son el medio principal para dar seguimiento a cosas como informes de errores, 
+solicitudes de mejoras o para informar otros problemas, como pruebas que fallan. 
+**No** están destinados a [solicitudes de soporte de usuarios]. Para ello, consulta la [guía de solución de problemas], 
+informa el problema en [Stack Overflow] o sigue en el [foro de Kubernetes].
+
+**Referencias:**
+- [Labels]
+- [Comandos de Prow][comandos]
+
+#### Crear un Issue
+
+- Utiliza una plantilla de issue si está disponible. Utilizar la correcta plantilla
+  ayudará a otros colaboradores a responder a tu issue.
+  - Sigue cualquier instrucción descrita en la plantilla del issue.
+- Sé descriptivo en el issue que estás planteando.
+- Asigna las [labels] adecuadas. Si tienes dudas, el bot [k8s-ci-robot][prow]
+  (bot [Kubernetes CI][prow]) responderá a tu issue con las etiquetas necesarias
+  para un triage correcto.
+- Sé selectivo al asignar issues usando [`/assign @<nombre de usuario>`][assign] o
+  [`/cc @<nombre de usuario>`][cc]. Tu issue se triageará de manera más efectiva aplicando
+  labels correctas en lugar de asignar más personas al issue.
+
+
+#### Responder a un Issue
+
+- Cuando estés abordando un issue, comenta en él para que otros sepan que estás trabajando en él
+  y evitar trabajo duplicado.
+- Cuando hayas resuelto algo por ti mismo en un momento posterior, comenta en el issue
+  para que la gente sepa antes de cerrarlo.
+- Incluye referencias a otros PRs o issues (o cualquier material accesible),
+  por ejemplo: _"ref: #1234"_. Es útil identificar que el trabajo relacionado se ha
+  abordado en otro lugar.
+
+
+### Abrir un Pull Request
+
+Las solicitudes de extracción (Pull Request o PR) son el principal medio 
+para contribuir con código, documentación u
+otras formas de trabajo que se almacenarán en un repositorio git.
+
+**Referencias:**
+- [Labels]
+- [Comandos de Prow][comandos]
+- [Proceso de Solicitud de Extracción (Pull Request)][Pull Requests]
+- [Flujo de Trabajo de GitHub]
+
+
+#### Crear un Pull Request
+
+- Sigue las instrucciones de la plantilla de solicitud de extracción si está disponible. 
```

Comment:
In the previous section (at [this line](https://github.com/kubernetes/community/pull/7559/files#diff-eb45794d5526e5387532ecacc84a222bbe0b0a660a5f87fee6ce264ba1d62ea0R176)) you have done the reference in parenthesis, so, I think we can use `PR` from now on, as you did in the following lines.

```suggestion
- Sigue las instrucciones de la plantilla de PR si está disponible. 
```